types/rdfjs__term-map/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for @rdfjs/term-map 1.0
+// Project: https://github.com/rdfjs-base/term-map
+// Definitions by: tpluscode <https://github.com/tpluscode>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Term } from "rdf-js";
+
+declare namespace TermMap {
```

Comment:
Why do you need this namespace.
You can just do this:
```ts
import { Term } from "rdf-js";
interface TermMap<T extends Term = Term, V = any> extends Map<T, V> {
}

// tslint:disable-next-line no-unnecessary-class
declare class TermMap<T extends Term = Term, V = any> {
    constructor(entries?: Array<[Term, V]>);
}
export = TermMap;
```

Reply:
Yes, I guess now it appears make less sense.

By adding the namespace it is easy to distinguish the imported class from the interface

```
import 
  TermMap, // this is the default-exported class
  { TermMap } // this is the interface from namespace export
from '@rdf-js/term-map'
```

Reply:
There’s no reason to distinguish between the interface and the class—that’s the whole point of declaration merging.